app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthFragment.kt

```diff
@@ -172,6 +177,8 @@ class BcscAuthFragment : Fragment(R.layout.fragment_bcsc_auth) {
             viewModel.resetAuthStatus()
             when (authStatus.tosAccepted) {
                 TOSAccepted.ACCEPTED -> {
+                    // clear timeline filter
```

Comment:
Why we are resetting filters in the BCSC auth ViewModel

Reply:
Removed from here